src/sparseml/transformers/sparsification/trainer.py

```diff
@@ -154,7 +156,7 @@ def __init__(
         self.criterion = torch.nn.CrossEntropyLoss()
         self.callback_disable_fp16 = DisableHalfPrecisionCallback(self)
         self.callback_handler.add_callback(self.callback_disable_fp16)
-
+        self._add_tensorboard_logger_if_available()
```

Comment:
Could you add TensorboardLogger to the list of loggers next to `[WANDBLogger()`] in the logger manager block above? Or is the issue that tensorboard is added to callbacks in some other piece of code, so you need to grab that instance from that list instead of instantiating a new one?

Reply:
we want to group it with some additional logic around the callback, so broke it into a separate function